### DISCUSSION TITLE
Slow Transfer Speeds

### DISCUSSION THREAD

**Discussion by lone-duck** [bdd100k/bdd100k#297] [Q&A]
Hello, 

I am aware that this is a duplicate of #287, but the issue seems to persist. Across multiple machines and both via wget and downloading from the browser, download speeds are on the order of 10KB/s, resulting in an eta of multiple years to download bdd100k_videos.zip. If anyone has any insight into this, or suggestions of how to download the dataset before my PhD is completed, it would be much appreciated :-).

**Comment by zyandtom** [comment]
hi do you find a solution?
